server/api/db/sqldb/db.py

```diff
@@ -3100,6 +3155,29 @@ def _re_tag_existing_object(
 
         return None
 
+    def _re_tag_existing_artifact(
```

Comment:
Why is this needed? There's no code in this PR that calls it?

Reply:
Leftover from some of my tests. Removing.